### ISSUE TITLE
Improve separator between splitted terminals

### ISSUE THREAD

**Issue by julienfalque** [gnome-terminator/terminator#329]
When disabling terminals titlebar, the separator between splitted terminals is barely visible on dark background:

![terminator](https://user-images.githubusercontent.com/1736542/102972267-11bcd480-44fb-11eb-8a95-8b7a235aeaf9.png)

I propose to improve this by either implementing a setting to change its color (and size maybe?) or make it automatically use a bright color when the background is dark.

I have no experience in Python but I'd be glad to give it a try with some guidance :)

**Comment by mattrose** [comment]
The best way is to run terminator in the GTK inspector.  There is probably a css property that you can tweak.

https://wiki.gnome.org/action/show/Projects/GTK/Inspector?action=show&redirect=Projects%2FGTK%2B%2FInspector

**Comment by julienfalque** [comment]
Thanks for your answer! I didn't know about this tool nor that using custom CSS files to tweak applications was possible!

I was able to improve things by creating the `~/.config/gtk-3.0/gtk.css` file with:
```css
.terminator-terminal-window separator {
  background: #303030;
}
```

Now that the separator is visible, I also noticed that the setting to change its size actually works (I thought it didn't because I didn't see any change when tweaking the setting).

Regarding the color, I'm wondering: why do I have a barely visible separator by default? Is it an issue with my global theme? Can something be improved in Terminator or should this be left to custom CSS files like I did?
